lab_4_summarization_textrank/main.py

```diff
@@ -193,15 +276,23 @@ def get_similarity_score(self, sentence: Sentence, other_sentence: Sentence) ->
         :param other_sentence
         :return: the similarity score
         """
-        pass
+        if sentence not in self._vertices or other_sentence not in self._vertices:
+            raise ValueError
+        idx1 = self._vertices.index(sentence)
```

Comment:
can it be erroneous? why?

Reply:
because index() method only returns the first occurrence of the value, no matter how many similar values are there